File: C#/PrimeiraWebAPI/PrimeiraWebAPI/Domain/Entity/Album.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;

namespace PrimeiraWebAPI.Domain.Entity
{
	[Table("Albuns")]
	public class Album
	{
		[Key]
		public int IdAlbum { get; set; }
		[Required]
		[StringLength(255)]
		public string Nome { get; set; }
		[Required]
		[StringLength(255)]
		public string Artista { get; set; }
		[Required]
		public int AnoLancamento { get; set; }

		public virtual ICollection<Avaliacao> Avaliacoes { get; set; }
	}
}

File: Java/olamundo/src/main/java/br/com/iteris/universidade/olamundo/domain/entity/Album.java
package br.com.iteris.universidade.olamundo.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.List;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Album {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int idAlbum;

    private String nome;

    private String artista;

    private int anoLancamento;

    @OneToMany(mappedBy = "album", fetch=FetchType.LAZY)
    private List<Avaliacao> avaliacoes;
}